packages/expo-camera/src/next/Camera.types.ts

```diff
@@ -311,7 +299,7 @@ export type CameraProps = ViewProps & {
    */
   mute?: boolean;
   /**
-   * Specify the quality of recorded video. Use one of [`VideoQuality.<value>`](#videoquality).
+   * Specify the quality of recorded video. Use one of `VideoQuality`.
    * Possible values: for 16:9 resolution `2160p`, `1080p`, `720p`, `480p` : `Android only` and for 4:3 `4:3` (the size is 640x480).
```

Comment:
```suggestion
   * Specify the quality of the recorded video. Use one of `VideoQuality` possible values:
   * for 16:9 resolution `2160p`, `1080p`, `720p`, `480p` : `Android only` and for 4:3 `4:3` (the size is 640x480).
```

It wasn't clear what "one of..." was so updated the sentence to make it clear that to use one of the possible values.